.eslintrc.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Remove this file and other files that are not related to this task

Reply:
@erickyvand ☝️

Reply:
These files were added when I did rebase from Develop, that is why I was asking do I need to delete it again?

And as you told me I'm waiting to rebase again